pkg/helm/release/manager_factory.go

```diff
@@ -86,13 +86,13 @@ func (f managerFactory) NewManager(cr *unstructured.Unstructured) (Manager, erro
 
 // getReleaseServer creates a ReleaseServer configured with a rendering engine that adds ownerrefs to rendered assets
 // based on the CR.
-func getReleaseServer(cr *unstructured.Unstructured, storageBackend *storage.Storage, tillerKubeClient *kube.Client) (*tiller.ReleaseServer, error) {
+func getReleaseServer(mgr crmanager.Manager, cr *unstructured.Unstructured, storageBackend *storage.Storage, tillerKubeClient *kube.Client) (*tiller.ReleaseServer, error) {
```

Comment:
Will `mgr` be needed for future field access? Perhaps a `meta.RESTMapper` parameter is enough if not.

Reply:
Thanks for pointing that out. Turns out I can get the `RESTMapper` from `tillerKubeClient`.